docs/06-troubleshooting.md

```diff
@@ -0,0 +1,114 @@
+We recommend using the [original Raspberry Pi power supply](https://www.raspberrypi.org/products/raspberry-pi-universal-power-supply/) and if not possible at least one that conforms to the power requirements described in [here](https://www.raspberrypi.org/documentation/hardware/raspberrypi/power/README.md).
+
+Thank you [@MatthewCroughan](https://github.com/MatthewCroughan) for the torough [investigation and testing](https://github.com/balenalabs/balena-sound/issues/62#issuecomment-605265537)!
+
+#### Rolling back to balenaOS 2.38 / Reduce Bluetooth UART baud rate
+Raspberry Pi 3's before rev 1.3 have no HW flow control on the UART that controlls the Bluetooth modem. This causes occasional data loss which result in the audio stuttering problem described above. Reducing the UART baud rate to `460800` lessens the problem significantly when compared to the default value of `921600`. 
+
+This however requires advanced knowledge and usage of balenaOS. So the advised change is to roll back to balenaOS version 2.38.0+rev1 which uses a default baud rate of `921600` but has other firmware conifiguration that avoids this problem. In order to roll back your balenaOS version you will need to re-flash your SD card and re-provision your device.
+
+We are currently working on bringing back the configuration present in balenaOS 2.38 to the latest version so that the roll back is not necessary; you can keep track of it [here](https://github.com/balena-os/balena-raspberrypi/issues/476).
+
+
+## No audio
+### Description
+
+No audio is coming out of your speaker system.
+
+### Solution
+Ensure that:
+- You correctly plugged an audio or HDMI cable to your device
+- You are connected to the device and streaming audio to it via any of the supported alternatives
+
+If you are using a DAC, make sure your DAC is on the supported list [here](https://balena-sound-preview-landr.netlify.app/docs/dac-configuration/) and that you applied the required configuration changes. If your DAC is not on the list, please let us know by opening an [issue](https://github.com/balenalabs/balena-sound/issues/new). We will work with you to find the correct configuration and add the DAC to the supported list.
+
+If you are using a USB soundcard, bear in mind that they are not *officially* supported by balenaSound. Some users have had positive results however not every Soundcard is the same. You can check out their progress here and please share your results whatever the outcome is:
+- https://github.com/balenalabs/balena-sound/issues/9#issuecomment-513283993
+- https://github.com/balenalabs/balena-sound/issues/9#issuecomment-544730189
+
+## Multi-room deviceas are out of sync
+### Description
+
+When using the multi-room feature there is a device or devices that are out of sync with the master server. This delay is usually a few hundred milliseconds, enough to be noticeable, and it doesn't change over time.
```

Comment:
Fix:

When using the multi-room feature, there might be devices that are out of sync with the master server. This delay is usually a few hundred milliseconds, enough to be noticeable, and it doesn't change over time.